### ISSUE TITLE
Weird GIF collection - Comment to have a GIF removed from the roleplay comments

### ISSUE THREAD

**Issue by ByteAlex** [ZeroTwo-Bot/ZeroTwo-Bot#78]

**Comment by xAngel04** [comment]
https://cdn.zerotwo.dev/PAT/ba1700a6-c6fe-4513-bd19-1845a20277fc.gif

**Comment by xAngel04** [comment]
Hug - https://cdn.zerotwo.dev/HUG/0d836f4a-1b95-4838-99ef-d8f4a9b4bdcd.gif

**Comment by xAngel04** [comment]
Cuddle - https://cdn.zerotwo.dev/CUDDLE/1e0b4367-b99d-42ac-a3df-2a5ea61650d4.gif

**Comment by xAngel04** [comment]
Lick - https://cdn.zerotwo.dev/LICK/cdd05f33-f9cd-4e1d-bf47-64ef4f010b98.gif

**Comment by xAngel04** [comment]
Wasted - https://cdn.zerotwo.dev/WASTED/cefb9f82-fb14-4d27-9d8e-c5bfde160130.gif

**Comment by xAngel04** [comment]
Wasted - https://cdn.zerotwo.dev/WASTED/04f8f44f-b367-4717-b85d-07d7cb00258e.gif

**Comment by xAngel04** [comment]
Stomp - https://cdn.zerotwo.dev/STOMP/8f6bb88a-96b5-4b96-9818-d46ddb2f0056.gif

**Comment by xAngel04** [comment]
Slap - https://cdn.zerotwo.dev/SLAP/9be3ae42-362a-42cb-a340-7bb73de67ff8.gif

**Comment by xAngel04** [comment]
Stomp - https://cdn.zerotwo.dev/STOMP/5feb5acb-dbce-4ddc-acf3-aee1ca864dfe.gif

**Comment by xAngel04** [comment]
Stomp - https://cdn.zerotwo.dev/STOMP/877c9e4d-b966-4baf-bd2a-9e25dadbe379.gif

**Comment by xAngel04** [comment]
Poke - https://cdn.zerotwo.dev/POKE/2ce02e9b-cfd2-4f01-86b9-f17259e0c16b.gif

**Comment by xAngel04** [comment]
Poke - https://cdn.zerotwo.dev/POKE/e2516895-bd09-41d5-8395-71c7b4205627.gif

**Comment by xAngel04** [comment]
Poke - https://cdn.zerotwo.dev/POKE/26c401db-85b5-45f3-bb91-45721e57c046.gif

**Comment by xAngel04** [comment]
Poke - https://cdn.zerotwo.dev/POKE/b644e7da-d8a5-49af-8e7f-b2aa0298c164.gif

**Comment by xAngel04** [comment]
Poke - https://cdn.zerotwo.dev/POKE/d25729c4-1dfe-4656-8770-89b3fde0748a.gif

**Comment by xAngel04** [comment]
Wasted - https://cdn.zerotwo.dev/WASTED/b6426ebd-6e54-48f3-839c-a0ffa46a6e53.gif

**Comment by xAngel04** [comment]
Wasted - https://cdn.zerotwo.dev/WASTED/f57a03fd-9eb3-49e0-979d-627d76f39eb7.gif

**Comment by xAngel04** [comment]
Bite - https://cdn.zerotwo.dev/BITE/4cdffd7c-4a52-4d32-9d9b-f9beef46f2fc.gif

**Comment by ByteAlex** [comment]
Unsure:
Hug - https://cdn.zerotwo.dev/HUG/0d836f4a-1b95-4838-99ef-d8f4a9b4bdcd.gif
Stomp - https://cdn.zerotwo.dev/STOMP/5feb5acb-dbce-4ddc-acf3-aee1ca864dfe.gif

These are for discussion

Disagree:
Wasted - https://cdn.zerotwo.dev/WASTED/f57a03fd-9eb3-49e0-979d-627d76f39eb7.gif
Poke - https://cdn.zerotwo.dev/POKE/d25729c4-1dfe-4656-8770-89b3fde0748a.gif
Poke - https://cdn.zerotwo.dev/POKE/b644e7da-d8a5-49af-8e7f-b2aa0298c164.gif
Poke - https://cdn.zerotwo.dev/POKE/26c401db-85b5-45f3-bb91-45721e57c046.gif
Wasted - https://cdn.zerotwo.dev/WASTED/cefb9f82-fb14-4d27-9d8e-c5bfde160130.gif

Whats wrong about these, I think they're matching actually well.

I agree about the others

**Comment by ruffrick** [comment]
Wasted - https://cdn.zerotwo.dev/WASTED/b6426ebd-6e54-48f3-839c-a0ffa46a6e53.gif
What's wrong about this one?

**Comment by ruffrick** [comment]
Lick - https://cdn.zerotwo.dev/LICK/b6195e51-38a4-4481-9581-d242967c9630.gif

**Comment by Test20C** [comment]
Bye - https://cdn.zerotwo.dev/GREET/165c77fc-12f0-4218-a59b-367c47883331.gif

**Comment by ByteAlex** [comment]
@Test20C https://cdn.zerotwo.dev/GREET/165c77fc-12f0-4218-a59b-367c47883331.gif
Is used in "GREET" it'd be fine if anyone can cut out the HIYA there, for now I'll remove it.
All other gifs marked with 🚀 were deleted

**Comment by ByteAlex** [comment]
Deleted gifs only apply to v2

**Comment by ByteAlex** [comment]
I removed this ticket from the project and cleared the milestone, but I'll leave it open for future removal requests.

**Comment by xAngel04** [comment]
> @Test20C https://cdn.zerotwo.dev/GREET/165c77fc-12f0-4218-a59b-367c47883331.gif
> Is used in "GREET" it'd be fine if anyone can cut out the HIYA there, for now, I'll remove it.
> All other gifs marked with  were deleted

If I remember right, I've seen this gif when I have used the bye command - multiple times.

**Comment by ByteAlex** [comment]
Yes, it is used in bye, because bye command is an alias to the GREET type

**Comment by BONBONauzWURST** [comment]
https://cdn.zerotwo.dev/PUNCH/8019644c-d37d-47cc-8956-567bb77eede5.gif
should be as kick not punch
"request send by an user on #help"
